test/ajax.test.js

```diff
@@ -31,6 +31,15 @@ describe('ajax', function() {
     })
   })
 
+  it ('automatically adds an Accept JSON header', function (done) {
+    ajax({
+      beforeSend(ajax) {
+        assert.equal(ajax.header.accept, 'application/json')
+        done()
```

Comment:
This test isn't asynchronous, but I like to use `done` any time I'm testing parameters that will be called by a function.